### PR TITLE
tweaks about Row type variable

### PR DESCRIPTION
**Pull request by dlax**

### CODE REVIEW

File: psycopg/psycopg/cursor.py [hunk 1]
```diff
@@ -781,7 +781,8 @@ def stream(
             first = True
             while self._conn.wait(self._stream_fetchone_gen(first)):
                 # We know that, if we got a result, it has a single row.
-                rec: Row = self._tx.load_row(0, self._make_row)  # type: ignore
+                rec = self._tx.load_row(0, self._make_row)
+                assert rec is not None
```
Comment (dvarrazzo) [inline psycopg/psycopg/cursor.py:785]: Actually I dropped this assert investigating in the stream() slowness (https://github.com/psycopg/psycopg/commit/562e8c46bbe5d2c93d4e4ac2203c7255d363c708). It wasn't caused by this assert of course, but I'm still mildly upset that Mypy checks affect runtime. I forgot to update the async side too.

What do you think? Two asserts or two type-ignore?

**Comment by dvarrazzo** [comment]
Thank you for this improvement. Is this change safe to be backported to the 3.0 branch?

File: psycopg/psycopg/cursor.py (hunk 1 above)
Comment (dlax) [reply psycopg/psycopg/cursor.py]: Ah, didn't noticed that; `# type: ignore` are fine I think, changed accordingly.

**Comment by dlax** [comment]
> Is this change safe to be backported to the 3.0 branch?

Yes, it should be; the commit applies fine to maint-3.0.

**Comment by dvarrazzo** [comment]
Thank you, that's neat :)

**Pull request merged**